### PR TITLE
[#9] WeatherIcon component

### LINKED ISSUE #9: Add weather icon component
- size
- weather title

### PR DESCRIPTION
**Pull request by inovux**
closes #9 

Added following functionality:

* Added app background color;
* Added mapper between weather icon names and image URLs;

I have added the following Properties:
* Added `size` property, which can either be `small` or `big` and defaults to `small`;
* Added `icon` property, which can be one of the names of the icons inside the `IMAGES` map;

Component is used like this:

```jsx
<WeatherIcon size="big" icon="hail" />
<WeatherIcon size="small" icon="heavyCloud" />
```

Screen Shot 2020-11-13 at 8 34 38 PM

### CODE REVIEW

**Review by mrpeterhanley — APPROVED** [review]
Looks good!

File: tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
+    "typeRoots": ["_typings"],
```
Comment (hananina) [inline tsconfig.json:4]: @inovux I assume that this line is importing a global type definition from `src/_typings/import-png.d.ts` correct? 😃

File: src/components/WeatherIcon/index.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react'
+import { IMAGES } from './WeatherIcons'
+
+interface IWeatherIcon {
+  size?: 'small' | 'big'
+  icon: keyof typeof IMAGES
```
Comment (hananina) [inline src/components/WeatherIcon/index.tsx:6]: 📝👀 is this setting the keys of the IMAGES object?

**Review by hananina — APPROVED** [review]
Looks Good to me🙌

File: tsconfig.json (hunk 1 above)
Comment (inovux) [reply tsconfig.json:4]: Yes correct :)

File: src/components/WeatherIcon/index.tsx (hunk 2 above)
Comment (inovux) [reply src/components/WeatherIcon/index.tsx:6]: It is actually basically getting all the keys from the IMAGES object. So when you use the icon property we know what keys / images are available as the values.

**Pull request merged**